src/style.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+}

```

Comment:
add `font-weight` and `font-style`
<img width="1017" alt="Screenshot 2023-04-18 at 17 44 51" src="https://user-images.githubusercontent.com/32853034/232831232-6c3c2ba2-cdae-474c-a913-5efa421b78f8.png">